recipes/welltestpy/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "welltestpy" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c010a8755e8f10a64492a06d5bc4c8be2587aefc437663a60ca126148b84b9e7
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools >=45
+    - setuptools_scm >=3.4
+  run:
+    - python
+    - anaflow >=1.0.0
+    - matplotlib-base >=3.0.0
+    - numpy >=1.14.5
+    - pandas >=0.23.2
+    - scipy >=1.1.0
+    - spotpy >=1.5.0
+
+test:
+  imports:
+    - welltestpy
+  requires:
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pytest -vv tests
+
+about:
+  home: "https://github.com/GeoStat-Framework/welltestpy"
+  license: MIT
+  license_family: MIT
+  license_file: 
```

A licence file is needed 
```suggestion
  license_file: LICENSE
```